packages/@aws-cdk/core/test/util.ts

```diff
@@ -1,6 +1,15 @@
-import { Stack } from '../lib';
+import * as cxapi from '@aws-cdk/cx-api';
+import { Stack, Construct, StackProps, App } from '../lib';
 import { synthesize } from '../lib/private/synthesis';
 
+export class TestStack extends Stack {
```

Comment:
I don't see this being used anywhere. Leftovers from an attempt to clean up the tests?

Reply:
Yes! Thank you.